src/components/EditorTopBar.tsx

```diff
@@ -46,11 +50,23 @@ export const EditorTopBar = ({
           <h2>{title}</h2>
         </li>
         <li>
-          {
-            children
-            // When children is empty or undefined, this li should be render anyway because it is on the right
-            // TODO: restructure header styles and ed-header-list class
-          }
+          <ul className="ed-header-list">
```

Comment:
I do not like this `ed-header-list` ul's class, but I understand that it is the way that we are doing this.